Playdown.swift

```diff
@@ -179,10 +181,25 @@ struct Playdown {
                 }
                 
             } catch {
-                print(error)
+                
+                //Handle case for single playground page or single swift file
```

Comment:
Rather than handle this in a catch, first check if the file handle you have is a directory and then act appropriately. Exceptions should be exceptional (e.g. they don't happen often).